portfolio/src/main/webapp/style.css

```diff
@@ -116,4 +127,12 @@ h6 {
   content: "";
   clear: both;
   display: table;
+}
+
+#myDIV {
+  width: 100%;
```

Comment:
Alphabetize declarations throughout this file: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
Done

Reply:
Done